src/Symfony/Component/Notifier/Tests/ChatterTest.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Symfony\Component\Notifier\Tests;
+
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Messenger\Envelope;
+use Symfony\Component\Messenger\MessageBusInterface;
```

Comment:
I guess you need to require the messenger component in require-dev to make the tests pass.

Reply:
I think we can use in `require-dev`:

`symfony/messenger: "^4.4 || ^5.0"`